Makefile

```diff
@@ -0,0 +1,234 @@
+# Makefile for Open Targets data pipeline
+# Based on steps in https://github.com/opentargets/data_release/wiki/OT011-Data-Processing-(Backend)
+
+# THhese variables can be overridden on the command-line
+ES_PREFIX = gp-makefile
+ELASTICSEARCH_NODES = http://localhost:9200
+LOG_LEVEL = DEBUG
+SCHEMA_VERSION = 1.2.8
+PERCENTAGE_TO_KEEP = 2
+
+# Internal variables
+MRTARGET_CMD = python -m mrtarget.CommandLine --log-level=$(LOG_LEVEL) $(ES_PREFIX)
+INDEX_CMD = python scripts/check_index.py --elasticsearch $(ELASTICSEARCH_NODES) --zero-fail --index
+DOWNLOAD_CMD = ./scripts/download_and_trim.sh
+
+# Main command for invoking mrtarget; note that $STAGE needs to be defined for this work.
+# Also note that $@ expands to the name of the current Makefile target so saves a lot of boilerplate
+TEMPLATE_CMD = $(MRTARGET_CMD) --$(STAGE) 2>&1 | tee out.$(ES_PREFIX).$(STAGE).log
```

Comment:
Can we add an ability to specify additional arguments to pass to stages e.g. `$(MRTARGET_ARGS)` ? Would be useful for the QC steps I'm working on.

Reply:
Good idea, commit 68f59a8 which I just pushed allows this.